### ISSUE TITLE
DOCSMD010/docsmd.xref erroneously triggered when link begins with xref but isn't an xref link

### ISSUE THREAD

**Issue by Thraka** [microsoft/vscode-docs-authoring#428]
I'm getting this error: `Bad xref syntax. xref should include a colon "<xref:System.Format.String%2A>"` on this content:

```markdown
For more information, see [x:Reference Markup Extension](xreference-markup-extension.md).
```

**Comment by bharney0** [comment]
@Thraka Thank you for reporting this issue. We have been able to repro, and will look into the problem more.

**Comment by bharney0** [comment]
PR for fixing issue implemented here #475
